WordPress/src/test/java/org/wordpress/android/viewmodel/pages/PagesViewModelTest.kt

```diff
@@ -233,4 +234,12 @@ class PagesViewModelTest {
 
         return pageModel
     }
+
+    private companion object Fixtures {
+        fun createPostModel() = PostModel().apply {
+            id = Random(100).nextInt()
```

Comment:
I wanted to use `Random` here in case the `createPostModel()` gets used in a `List`. But I see the error in my ways. There's a bug because the seeded `Random` is inside the method. ┻━┻ ︵ヽ(`Д´)ﾉ︵﻿ ┻━┻

Thank you for sharing your wisdom. 🙂 I removed it for now in 8f4f7e37304d0ecd5f015d8569c9b99392c05582 since it's overkill.